vtb3_5.m

```diff
@@ -109,7 +113,7 @@ function vtb3_5(a0,aodd,aeven,bodd,beven,N,T,o)
 	end
 	%dt=max([dt T/10000])
 	t=0:dt:T*3;length(t);
-	F=0*t+eval(a0)/2;
+	F=0*t+eval(a0);
```

Comment:
a) Note that the toolbox uses Inman's definition
b) `t=0:.001:1;f = 0*t+1; vtb3_3(f,t,15);` 
results in a correct plot. What is the error? Provide an example. 

I like the example, but I cannot accept it because you lumped it together with this.